test/check/check.go

```diff
@@ -75,6 +75,61 @@ func CheckComponentsRemoval(components []Component) {
 	}
 }
 
+func getConfigComponentsMap(gvk schema.GroupVersionKind) (map[string]struct{}, error) {
```

Comment:
at golang sets are usually expressed as `map[string]bool`

Reply:
There is no need for a bool, I only want to check if the component is "set" or not. 
IIUC `map[string]struct{}` is idiomatic and preferred in Go when you don't need to store a value, only the presence of a key.

Reply:
i also think bool is better

Reply:
```
if _, exist := deployedComponents[component.ComponentName]; !exist {
   foo()
}
```

semantic bool is easier to read imho

Reply:
yeah the boolValue wan't really needed I just wanted to make it more obvious..

Well then I guess it's personal preference. I like struct{} cause it saves memory space, but again it's not super important.